pkg/cmd/api/api.go

```diff
@@ -276,7 +279,7 @@ func NewCmdApi(f *cmdutil.Factory, runF func(*ApiOptions) error) *cobra.Command
 	cmd.Flags().StringArrayVarP(&opts.MagicFields, "field", "F", nil, "Add a typed parameter in `key=value` format")
 	cmd.Flags().StringArrayVarP(&opts.RawFields, "raw-field", "f", nil, "Add a string parameter in `key=value` format")
 	cmd.Flags().StringArrayVarP(&opts.RequestHeaders, "header", "H", nil, "Add a HTTP request header in `key:value` format")
-	cmd.Flags().StringSliceVarP(&opts.Previews, "preview", "p", nil, "GitHub API preview `names` to request (without the \"-preview\" suffix)")
+	cmd.Flags().StringSliceVarP(&opts.Previews, "preview", "p", nil, "Opt into GitHub API previews (names should exclude '-preview' and be comma separated)")
```

Comment:
Like other slices, this can also be provided like so:

```
gh api --verbose --preview foo --preview bar
```

I'm not sure it's necessary to mention the accepted format here when we don't do it for any other flag.

Reply:
I didn't ask for it, but I was surprised to learn it was a thing, without this hint I wouldn't have known. 

I'm not attached to that part of the change. 

I agree with your suggested change to the start. I was just copy+pasting the suggestion.

Reply:
@williammartin that was meant to address one of the example usages where the previews are comma-separated. I don't think it's a big deal, though.